src/ui_fsmenu/multiplayersetupgroup.cc

```diff
@@ -292,17 +280,15 @@ struct MultiPlayerPlayerGroup : public UI::Box {
 
 		// Init dropdowns
 		update();
-		layout();
 	}
 
-	/// Update dropdown sizes
-	void layout() override {
-		const int max_height = g_gr->get_yres() * 3 / 4;
-		type_dropdown_.set_height(max_height);
-		tribes_dropdown_.set_height(max_height);
-		init_dropdown_.set_height(max_height);
-		team_dropdown_.set_height(max_height);
-		UI::Box::layout();
+	void force_new_dimensions(float, uint32_t max_width, uint32_t standard_element_height) {
```

Comment:
Is there a reason you renamed this function? I have been calling this `layout()` everywhere so far, so having a different name here will make the code harder to read IMO.

Reply:
Yeah, that was intentional. I wanted to highlight that these panels cannot relayout themselves but need parameters from "outside". I can rename them to layout(float, uint32_t, ...) if you like. Not sure if this actually improves readability because this of course would not override Panel::layout(), right? ;)

Reply:
Duh, I overlooked that.